src/mockServiceWorker.js

```diff
@@ -158,10 +163,11 @@ self.addEventListener('fetch', function (event) {
 
       switch (clientMessage.type) {
         case 'MOCK_SUCCESS': {
-          setTimeout(
-            resolve.bind(this, createResponse(clientMessage)),
-            clientMessage.payload.delay,
-          )
+          const t = setTimeout(() => {
```

Comment:
@kettanaito what was the reason to add the timeout in this service worker?
I was thinking some abort logic could be centralized if the delay would've  been handled in the library.

Reply:
The timeout was added here to push the delaying as much up as possible (service worker being the last link in the request handling chain). We should be able to move the delay to the library's code, but we need to inspect how that delay will be reflected in the Network tab. I have a suspicion there's a chance the response time will be instant if the delay is on the library side, but I'm not sure.